torch/csrc/jit/tensorexpr/half_support.h

```diff
@@ -1,14 +1,15 @@
 #pragma once
 
-#include <torch/csrc/jit/codegen/fuser/cuda/resource_strings.h>
```

Comment:
Cuda codegen needs this. You'll have to add it back there.

Reply:
This breaks the build when you're not using cuda... Why does it need to be in this file?